benchmarking/driver/benchmark_driver.py

```diff
@@ -36,17 +37,18 @@ def runOneBenchmark(info, benchmark, framework, platform, backend, reporters, lo
             data = _mergeDelayData(data, control, bname)
         if benchmark["tests"][0]["metric"] != "generic":
             data = _adjustData(info, data)
-        meta = _retrieveMeta(info, benchmark, platform, framework, backend)
+            meta = _retrieveMeta(info, benchmark, platform, framework, backend)
 
         result = {
             "meta": meta,
             "data": data
         }
-    except Exception:
+    except Exception as e:
         # Catch all exceptions so that failure in one test does not
         # affect other tests
         getLogger().info(
             "Exception caught when running benchmark")
+        getLogger().info(e)
```

Comment:
put the two logs together.

Reply:
I prefer to print the exception in a separate line. This may be clearer even though it may seem verbose. What is your suggestion?